Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -71,6 +72,23 @@ struct EFBCopyParams
   bool copy_filter;
 };
 
+template <>
+struct fmt::formatter<EFBCopyParams>
+{
+  constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+  template <typename FormatContext>
+  auto format(const EFBCopyParams& uid, FormatContext& ctx)
+  {
+    std::string_view copy_format;
+    if (uid.copy_format == EFBCopyFormat::XFB)
+      copy_format = "XFB";
+    else
+      copy_format = fmt::to_string(uid.copy_format);
```

Comment:
This seems unsafe: `copy_format` is a string_view, and fmt::to_string returns a std::string which is destructed after the assignment, so I think `copy_format` would end up being a dangling view.